manager/allocator/networkallocator/networkallocator.go

```diff
@@ -657,7 +658,12 @@ func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, str
 }
 
 func (na *NetworkAllocator) loadDriver(name string) error {
-	_, err := plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	var err error
+	if pg := na.drvRegistry.GetPluginGetter(); pg != nil {
+		_, err = pg.Get(name, driverapi.NetworkPluginEndpointType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	}
```

Comment:
What are the semantics of doing it like this? Does it mean that if we have a `PluginGetter`, we only support v2 plugins, and without a `PluginGetter`, only v1 plugins are supported? Or is there some compatibility mechanism built into `pg.Get`?